src/components/slideshow/slideshow.js

```diff
@@ -344,6 +381,9 @@ export default function (options) {
             slides = currentOptions.items;
         }
 
+        const autoplayDelay = userSettings.get('autoplayDelay', true) || 3000;
+        const autoplay = options.autoplay ? {delay:autoplayDelay} : false;
```

Comment:
_Not tested_
```suggestion
        const autoplay = !options.interactive || !!options.autoplay ? { delay: autoplayDelay } : false;
```
But I was also thinking about specifying a delay in the options.

Reply:
Done. I looked into a bit about doing it with options, but I'm not familiar enough with the project to know the best way to do it. The current way seems fine to me for now.